vcd/resource_vcd_org_vdc.go

```diff
@@ -269,7 +274,7 @@ func resourceVcdVdcCreate(d *schema.ResourceData, meta interface{}) error {
 	adminVdc, err := adminOrg.GetAdminVDCByName(d.Get("name").(string), false)
 	if err != nil {
 		log.Printf("[DEBUG] Unable to find vdc.")
-		return fmt.Errorf("unable to find vdc.. %#v", err)
+		return fmt.Errorf("unable to find VDC. %#v", err)
```

Comment:
No need for interface here, it doesn't show more information, just wraps it with additional structures which is less user friendly.
```suggestion
		return fmt.Errorf("unable to find VDC. %s", err)
```

Reply:
fixed